vscode-wpilib/locale/zh-cn/package.i18n.yaml

```diff
@@ -9,6 +9,7 @@ wpilibcore.simulateCode.title: 在电脑上模拟运行机器人代码
 wpilibcore.setLanguage.title: 设置语言
 wpilibcore.setAutoSave.title: 设置部署时自动保存
 wpilibcore.setSkipTests.title: 设置是否跳过测试API
+wpilibcore.selectDefaultSimulateExtension.title: Set to select the simulation extension automatically, if there is only one
```

Comment:
I added this temporary string in lieu of a proper translation, because I'm unsure of whether it's better to leave the temporary English string or omit it entirely.

Reply:
Temp English string is definitely the way to go here.